packages/tuxguitar/tuxguitar.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+export PATH="/usr/lib/jvm/java-11-openjdk/bin:$PATH"
+CP=/usr/share/tuxguitar/dist
+CP="$CP:/usr/share/doc/tuxguitar"
```

Comment:
Minor niggle: combine these two lines

Reply:
done in b6d7aa487dbbb94e8837878bfafeda0f27f20c3f